x-pack/plugins/security_solution/server/endpoint/lib/artifacts/lists.ts

```diff
@@ -140,8 +140,6 @@ export async function getEndpointEventFiltersList(
     policyId ? ` or exception-list-agnostic.attributes.tags:\"policy:${policyId}\"` : ''
   })`;
 
-  await eClient.createEndpointEventFiltersList();
-
   return getFilteredEndpointExceptionList(
```

Comment:
Just a question: I was wondering if this here would not `throw`, so I followed this code from here and it looks like when the list does not exists, that `null` is just returned which causes this method call to return an empty array (`{ entries: [] }`). Do I have that right?

Reply:
Exactly, this is the artifact for event filters when the list doesn't exists yet:
```
-------------------------------------------------------------------
Policy:   test
Manifest: 1.0.0 | v1
Artifact: endpoint-eventfilterlist-windows-v1
          Relative URL:   /api/fleet/artifacts/endpoint-eventfilterlist-windows-v1/d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658
          Encoded SHA256: f8e6afa1d5662f5b37f83337af774b5785b5b7f1daee08b7b00c2d6813874cda
          Decoded SHA256: d801aa1fb7ddcc330a5e3173372ea6af4a3d08ec58074478e85aa5603e926658
-------------------------------------------------------------------

{
  "entries": []
}

-------------------------------------------------------------------
```